Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -325,6 +335,8 @@ def get_X_and_y_from_data(data, text_field):
 
 
 def main():
+    tokenizer_script = demisto.args().get('tokenizerScript', None)
```

Comment:
what is `tokenizerScript`?
1. If it's aimed for the user to change, you need to expose it in the yml 
2. if same argument exists in other automations, try to be consistent with it

Reply:
@lperryp it's like the tokenizer method, we have only 1 now. 
It's suppose to be part of the script inputs, @yguezpa please add this with default value of "nlp", also you can change the name of the argument to be "tokernizerMethod"